meetings/2020-04-01.md

```diff
@@ -0,0 +1,96 @@
+﻿# Node.js Technical Steering Committee (TSC) Meeting 2020-04-01
+
+## Links
+
+* **Recording**:  
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/842
+
+
+## Present
+
+
+* Anna Henningsen @addaleax (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Gireesh Punathil @gireeshpunathil (TSC)
+* James Snell @jasnell (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Matheus Marchini @mmarchini (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawso* Ruben Bridgewater @BridgeAR (TSC)

```

Comment:
This line is off. And Ruben is included below as well.

Reply:
@cjihrig fixed